homeassistant/components/climacell/strings.json

```diff
@@ -3,12 +3,12 @@
   "config": {
     "step": {
       "user": {
-        "description": "If [%key:component::climacell::config::step::user::data::latitude%] and [%key:component::climacell::config::step::user::data::longitude%] are not provided, the default values in the Home Assistant configuration will be used. An entity will be created for each forecast type but only the ones you select will be enabled by default.",
+        "description": "If [%key:common::config_flow::data::latitude%] and [%key:common::config_flow::data::longitude%] are not provided, the default values in the Home Assistant configuration will be used. An entity will be created for each forecast type but only the ones you select will be enabled by default.",
         "data": {
           "name": "Name",
```

Comment:
```suggestion
          "name": "[%key:common::config_flow::data::name%]",
```

Reply:
😅

Reply:
Now the title of the PR is wrong. 😄 
By the way thanks for the integration.

Reply:
fixed! and no problem!